lib/vfscore/automount.c

```diff
@@ -900,5 +906,10 @@ err_umount: __maybe_unused
 	vfscore_autoumount(NULL);
 	return rc;
 }
+#endif /* CONFIG_LIBVFSCORE_AUTOMOUNT */
 
+#if CONFIG_LIBVFSCORE_AUTOMOUNT && CONFIG_LIBVFSCORE_AUTOUNMOUNT
 uk_rootfs_initcall_prio(vfscore_automount, vfscore_autoumount, 4);
+#elif CONFIG_LIBVFSCORE_AUTOUNMOUNT
+uk_rootfs_initcall_prio(0x0, vfscore_autoumount, 4);
+#endif /* CONFIG_LIBVFSCORE_AUTOUNMOUNT */
```

Comment:
`/* !CONFIG_LIBVFSCORE_AUTOMOUNT && CONFIG_LIBVFSCORE_AUTOUNMOUNT */`?

Reply:
It is this way because of the `#else` case.
We had agreed on:
```c
#if COND_A

#else /* !COND_A - expression that is covering all previously non-catched cases */

#endif /* !COND_A - repeat last condition */
```

and
```c
#if COND_A

#elif COND_B

#endif /* !COND_B - repeat last condition */
```